src/pages/authenticationPage/Signup.js

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useState } from "react";
+import "./auth.css";
+import { Link, useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { signupHandler } from "../../redux/reducer/authenticationSlice";
+
+const Signup = () => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const { token } = useSelector((store) => store.authentication);
+  const [error, setError] = useState("");
+  const [userCredentials, setUserCredentials] = useState({
+    email: "",
+    password: "",
+    confirmPassword: "",
+    firstName: "",
+    lastName: "",
+  });
+
+  const sumbitHandler = (e) => {
+    e.preventDefault();
+    if (
+      userCredentials.password.length >= 9 &&
```

Comment:
I liked that you enforced a length constraint.

Reply:
Thank You